src/Components/Artwork/GridItem.tsx

```diff
@@ -120,6 +122,7 @@ export const ArtworkGridItem: React.FC<ArtworkGridItemProps> = ({
           disableRouterLinking={disableRouterLinking}
           to={to}
           renderSaveButton={renderSaveButton}
+          saveOnlyToDefaultList={saveOnlyToDefaultList}
```

Comment:
Really wish we just set up a simple artwork grid provider to avoid the props drilling